apps/settings/elements/wifi_enter_certificate_nickname.html

```diff
@@ -2,7 +2,7 @@
   <template>
 
     <gaia-header>
-      <button type="reset" data-icon="back"><span data-l10n-id="back">Back</span></button>
+      <button type="reset"><span data-icon="close"><span data-l10n-id="back">Back</span></span></button>
```

Comment:
I thought we have to leave a span with text for the screen reader. Although I found this has not be done in many panels. We can't use the same span because the text will be displayed along with the icon.